site.yml

```diff
@@ -60,3 +60,30 @@
   - { role: swarm, run_as: worker }
   - { role: contiv_network, run_as: worker }
   - { role: contiv_storage, run_as: worker }
+
+# net-master hosts correspond to cluster machines that run the master
+# logic of the contiv net services
+- hosts: net-master
+  sudo: true
+  environment: env
+  roles:
+  - { role: base }
```

Comment:
"To put it the other way around, if you use net-master host-group for provisioning a vagrant environment it would run base role twice, right?"

It is not intent of this PR to use net-master host group for provisioning vagrant. However, it can be the intent of another PR.